test/integration/dialects/postgres/query.test.js

```diff
@@ -135,5 +135,44 @@ if (dialect.startsWith('postgres')) {
         },
       });
     });
+
+    it('orders by a literal when subquery and minifyAliases are enabled', async () => {
+      const sequelizeMinifyAliases = Support.createSequelizeInstance({
+        logQueryParameters: true,
+        benchmark: true,
+        minifyAliases: true,
+        define: {
+          timestamps: false,
+        },
+      });
+
+      const Foo = sequelizeMinifyAliases.define('Foo', {
+        name: {
+          field: 'my_name',
+          type: DataTypes.TEXT,
+        },
+      }, { timestamps: false });
+      await sequelizeMinifyAliases.sync({ force: true });
+
```

Comment:
My inner monk asks if you could move this blank line up so that all the await statements are nicely aligned in its own block